targets/TARGET_Cypress/scripts/mbed_set_post_build_cypress.cmake

```diff
@@ -23,13 +23,15 @@ function(mbed_post_build_psoc6_merge_hex mbed_target_name)
             COMMAND ${Python3_EXECUTABLE} ${MBED_PATH}/targets/TARGET_Cypress/scripts/PSOC6.py
                 merge
                 --artefacts-location ${CMAKE_BINARY_DIR}
+                --application-name $<TARGET_PROPERTY:mbed-post-build-bin-${mbed_target_name},application>
```

Comment:
You could just get rid of  `artefacts-location` and pass the hex and elf directly. This way we won't have to build the path to the file in the Python script.

Reply:
Done, I've brought back `--m4hex` and `--elf`.